### DISCUSSION TITLE
gituhb page shows 404 error

### DISCUSSION THREAD

**Discussion by williamkwon97** [community/community#21813] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/665758ee08e23565b9c9485c55ba9420de0204d1.png" data-download-href="/uploads/short-url/eBlQg41Nzg2fI86v7w8YYcbx36F.png?dl=1" title="Screen Shot 2020-02-14 at 7.59.50 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/34716439/181066992-a3f2a262-9860-45d8-a661-21154b72b3a8.png" alt="Screen Shot 2020-02-14 at 7.59.50 AM.png" data-base62-sha1="eBlQg41Nzg2fI86v7w8YYcbx36F" width="690" height="283" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/665758ee08e23565b9c9485c55ba9420de0204d1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-02-14 at 7.59.50 AM.png</span><span class="informations">691×284 17.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>It shows correct website for few seconds then it redirect to 404 error webpage</p>
<p><a href="https://williamkwon97.github.io" rel="nofollow noopener">https://williamkwon97.github.io</a> to <a href="https://williamkwon97.github.io/public/index.html" rel="nofollow noopener">https://williamkwon97.github.io/public/index.html</a></p>
<p><a href="https://github.com/williamkwon97/williamkwon97.github.io" rel="nofollow noopener">https://github.com/williamkwon97/williamkwon97.github.io</a></p>

**Comment by logankilpatrick** [comment]
<p>Hey! This is the issue: </p>
<pre><code>https://github.com/williamkwon97/williamkwon97.github.io/blob/5b4d6d89c98df9736aead25d90f9cbaea6876802/index.html#L2
</code></pre>
<p>See here: <a href="https://www.w3schools.com/tags/att_meta_http_equiv.asp" rel="nofollow noopener">https://www.w3schools.com/tags/att_meta_http_equiv.asp</a> for more info.</p>
<p>TL;DR you are re-directing the user after 2 seconds to a page that does not exist. </p>

**Comment by williamkwon97** [comment]
<p>I still have a same issue after I delete that line.</p>
<p>I added that line to fix this problem.</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Are you sure you gave it enough time to show you the updated site? I just tested your website and it seems to work. </p>

**Comment by logankilpatrick** [comment]
<p>I refreshed a few times and it is still working for me.</p>

**Comment by williamkwon97** [comment]
<p>Thanks It works now</p>

**Comment by myange111** [comment]
<p>please help me </p>
<p>im trying to get in touch with the board of directors about a very big issue concerning a member of staff targeting me and blocking my access</p>
<p>please pass the message on as i have had no reply as everything is blocked can u ask them to access my email account and leave any info they wish me to have in junk and i will be so gratful, i trully dont know what to do im stuck in this where i get no mail no call no one can contact me and its so painful</p>
<p>please help</p>
<p>yours faithfully</p>
<p>mrs angela tayler</p>
